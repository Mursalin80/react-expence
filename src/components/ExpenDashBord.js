import React from "react";
import ExpenList from "./ExpenList";
import ExpenListFilter from "./ExpenListFilter";
import AddExpence from "./AddExpence";

export default function ExpenDashBord() {
  return (
    <div>
      <h1>Dash Bord</h1>
      <AddExpence />
      <ExpenListFilter />
      <ExpenList />
    </div>
  );
}
