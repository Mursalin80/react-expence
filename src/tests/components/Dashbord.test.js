import { shallow } from "enzyme";
import React from "react";
import toJson from "enzyme-to-json";
import "../Enyzme.config";
import ExpenDashBord from "../../components/ExpenDashBord";

test("should render ExpenDashBord page", () => {
  let wrap = toJson(shallow(<ExpenDashBord />));
  expect(wrap).toMatchSnapshot();
});
