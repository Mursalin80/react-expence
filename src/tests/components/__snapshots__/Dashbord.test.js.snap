// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenDashBord page 1`] = `
<div>
  <h1>
    Dash Bord
  </h1>
  <Connect(ExpenListFilter) />
  <Connect(ExpenList) />
</div>
`;
